### PR TITLE
🔧(project) improve production swarm compose and documentation

### PR DESCRIPTION
**Pull request by wilbrdt**
## Purpose

The documentation and the Docker Swarm production compose file were missing key
aspects to correctly deploy Warren-TdBP.

## Proposal

- Fixing network
- Fixing databases creation
- Fixed dev logging file
- Expand and improve deployment instructions in README

### CODE REVIEW

File: production/README.md [hunk 1]
```diff
@@ -1,93 +1,189 @@
...
-Warren-TdBP `app` and `api` services need a PostgreSQL database for persistency. We need
-to create a Docker volume for the database, but also for the static and media files
-(required by the `app` service).
+Warren-TdBP requires a PostgreSQL database for persistency, along with volumes for
+static and media files used by the `app` service.
 
-In this tutorial, we'll create three volumes:
+Create the necessary volumes using the following commands:
 ```bash
 docker volume create postgresql_data
```
Comment (AmelieSOUBRAT) [inline production/README.md]: replace postgresql_data with postgres_data

File: production/docker-compose.prod.yml [hunk 2]
```diff
@@ -85,3 +92,7 @@ volumes:
     external: true
   static:
     external: true
+
+networks:
+  backend:
+    external: true
```
Comment (wilbrdt) [inline production/docker-compose.prod.yml]: Missing EOF CR

File: production/README.md [hunk 3]
```diff
@@ -1,93 +1,189 @@
...
+
+To make Warren-TdBP accessible to an external Learning Management System (LMS), we will
+utilize [Caddy](https://caddyserver.com/docs/install) as a reverse proxy. While Caddy
+can be launched via its CLI, we strongly recommend employing a service manager for
+automatic restarts (refer to the [documentation](https://caddyserver.com/docs/running)).
+
+Begin by creating a Caddyfile which contains your configuration:
+```
+app.example.com
+reverse_proxy :8080
```
Comment (wilbrdt) [inline production/README.md]:
```suggestion
reverse_proxy :8000
```

File: production/README.md [hunk 4]
```diff
@@ -1,93 +1,189 @@
...
+```
+
+Caddy is now operational, directing traffic to your pods.
+
+### Configuring Warren-TdBP for LTI integration
+
+With Warren-TdBP running, let's configure it as an external LTI tool within our chosen
+LMS. Start by creating an admin account on the Warren-TdBP app using:
+```bash
+docker exec -it $(docker ps -q -f name=tdbp_app) pythonS manage.py createsuperuser
```
Comment (wilbrdt) [inline production/README.md]:
```suggestion
docker exec -it $(docker ps -q -f name=tdbp_app) python manage.py createsuperuser
```

File: production/README.md [hunk 5]
```diff
@@ -1,93 +1,189 @@
...
+can be launched via its CLI, we strongly recommend employing a service manager for
+automatic restarts (refer to the [documentation](https://caddyserver.com/docs/running)).
+
+Begin by creating a Caddyfile which contains your configuration:
+```
+app.example.com
+reverse_proxy :8080
+
+api.example.com
+reverse_proxy:8100
```
Comment (wilbrdt) [inline production/README.md]:
```suggestion
app.example.com {
reverse_proxy :8080 }

api.example.com {
reverse_proxy :8100 }
```
Also: need to have "{}" when multiple entries

**Review by AmelieSOUBRAT — APPROVED** [review]

**Review by apui-avignon — APPROVED** [review]

**Pull request merged**